content/3.community/1.wallet.md

```diff
@@ -51,50 +68,33 @@ It is the only way to recover your account if you ever forget your password.
 ::
 ::alert{variant="success"}
 After securing your mnemonic, you can delete your bash history to ensure no one can retrieve it
-::highlight-card
 
 ```bash
 history -c
 rm -rf ~/.bash_history`
 ```
 
-::
 #title
 Tip
 ::
 
-### Using the Developer CLI
+## Using the Developer CLI
 
-Creating an account with the developer CLI is similar to using the daemon (`archwayd`), but has the benefit of making it easy to switch between Dockerized and native built versions of Archway core.
+Creating an account with the developer CLI is similar to using the daemon (`archwayd`), but has the benefit of making it easy to switch between Dockerized and native-built versions of Archway core. You can look at the archway CLI as a wrapper of `archwayd`, with additional functionalities for smoother use.
 
-[Read the developer instructions for creating a wallet](/developers/getting-started/setup#creating-an-account)
 
-### Using a Ledger device (hardware wallet)
+1. [install the archway CLI](/developers/getting-started/install#archway-developer-cli)
 
-When you initialize your Ledger, a 24-word mnemonic is generated and stored in the device. This mnemonic is compatible with Cosmos and Cosmos accounts can be derived from it. The following steps help make your Ledger compatible with the Archway daemon.
 
-1. Download the _Ledger Live_ app
-2. Connect your Ledger via USB and update to the latest firmware
-3. From the _Ledger Live_ app store, download the _Cosmos_ application. You may have to enable _Dev Mode_ in the settings of _Ledger Live_ to be able to download it.
-4. Navigate to the _Cosmos_ app on your Ledger device
-5. Create your account with the `archwayd` daemon using the flag `--ledger`
 
-::highlight-card
+2. [Create a wallet with the Archway CLI](/developers/getting-started/setup#creating-an-account)
 
-```bash
-archwayd keys add ${ACCOUNT_LABEL} --ledger
-```
 
-::
-::alert{variant="error"}
-Creating Ledger accounts will only work if the Ledger is charged, plugged in and unlocked.
-#title
-Caution
-::
+## Using a Browser Extension (Keplr)
 
-### Using a Browser Extension (Keplr)
+As previously stated, you will be able to derive your Archway address from your Cosmos address once Archway's mainnet is launched. 
```

Comment:
I don't think we should be saying this as whichever account is created within Keplr will generate the respective address for any chain that is connected to Keplr.
```suggestion
As previously stated, you will be able to derive your Archway address from your Cosmos address once Archway's mainnet is launched. 
```